src/main/java/com/exasol/releaserobot/repository/maven/MavenPom.java

```diff
@@ -23,6 +28,15 @@ public String getArtifactId() {
         return this.artifactId;
     }
 
+    /**
+     * Check if artifact id presents.
+     * 
+     * @return true if artifactId presents
+     */
+    public boolean hasArtifactId() {
```

Comment:
I see that the `getArtifactId` method is used, but it looks like this value might not be always here. Could there be any problem?